### PR TITLE
0.25

### PR DESCRIPTION
**Pull request by mxcl**

### CODE REVIEW

File: src/app.magic.ts [hunk 1]
```diff
@@ -21,8 +21,10 @@ export default function(self: Path, shell?: string) {
         chpwd_functions=( _tea_chpwd_hook \${chpwd_functions[@]} )
       fi
 
-      # if the user put tea in eg. /usr/local/bin then don’t pollute their PATH
-      # we check for \`tea --prefix\` due to \`gitea\` being \`tea\` when installed with \`brew\`
+      if [[ "$PATH" != *"$HOME/.local/bin"* ]]; then
```
Comment (jhheider) [inline src/app.magic.ts:24]: There's an extremely minor edge-case here, where this will skip if they have `$HOME/.local/bin/foo` in their path. A better test (with more dependencies) is `echo $PATH | tr ':' '\n' | grep -E "^$HOME/.local.bin$"`

Comment (jhheider) [reply src/app.magic.ts:24]: Or `split` as you do below.

Comment (mxcl) [reply src/app.magic.ts:24]: You’re right. I think this will have to do for now though as really trying to only use builtins for speed and portability.

**Comment by jhheider** [comment]
Well, that wasn't intended. I'll back all this up.
